app/assets/javascripts/discourse/app/templates/composer/dominating-topic.hbs

```diff
@@ -0,0 +1,13 @@
+<a href {{action closeMessage message}} class="close">{{d-icon "times"}}</a>
+<div class="dominating-topic-content">
+  {{html-safe message.body}}
```

Comment:
I was talking specifically of wrapping this line, or is it already generating a <p></p> ?